packages/yew-router/tests/browser_router.rs

```diff
@@ -112,19 +114,29 @@ fn root() -> Html {
 // - query parameters
 // - 404 redirects
 #[test]
-fn router_works() {
+async fn router_works() {
     yew::start_app_in_element::<Root>(gloo::utils::document().get_element_by_id("output").unwrap());
 
+    sleep(Duration::ZERO).await;
```

Comment:
This is used to often, it should get a name in the test code like `retire_scheduled_changes()` to clarify its purpose.

Reply:
sleep(0) is universally used in the context of yielding the control from a task and schedule the resumption of itself to the end of the event loop. There's no need to clarify the purpose of it to people who're familiar with asynchronous IO. And it's more confusing if an alias is used as I have to see the content of the alias to determine what it does.